repository/CargoPackageManager-Calypso-Browser/PBCreatePackageInProjectCommand.class.st

```diff
@@ -48,20 +48,20 @@ CGOCreatePackageInProjectCommand class >> systemBrowserToolbarActivatorForPackag
 ]
 
 { #category : #execution }
-CGOCreatePackageInProjectCommand >> createSystemPackage [
+PBCreatePackageInProjectCommand >> createSystemPackage [
 	SycAddNewPackageCommand new
 		systemEnvironment: toolContext systemEnvironment;
```

Comment:
Normally commands should not reference context. They should be self contained objects and extract required collaborators from context. Here it means that command should have variable #systemEnvironment.
But I did not look deeply. In some cases keeping context is suitable workaround.

Reply:
I don't see a reason against nor in favour. Can you elaborate?